QualityControl/public/common/filters/runMode/runStatusPanel.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { h } from '/js/src/index.js';
+
+/**
+ * Render a run mode switch
+ * @param {number} runNumber - The run number
+ * @param {RunStatus} status - The run status
+ * @returns {HTMLElement} - The rendered run status panel
+ */
+export const runStatusPanel = (runNumber, status) =>
+  status.match({
+    Loading: () =>
+      h('div.flex-row.g1.items-center.justify-center', [
+        h('b', `#${runNumber}`),
+        h('div.flex-row.g1', [
+          h('div', { class: 'label' }, 'Status: '),
```

Comment:
Why dedicated `class` when you can use the `.label` like you do a line below and above for classes?